app/components/views/Offset/index.tsx

```diff
@@ -352,11 +352,12 @@ export const Offset = (props: Props) => {
       } else if (paymentMethod === "fiat") {
         return; // type guard
       } else {
+        const maxAmountIn = allowances?.[paymentMethod];
```

Comment:
Unfortunately we can't rely on this because some people might have infinite allowances from early days :(

Need to rely on the quoted cost. I'm having analysis paralysis about adding slippage buffer or not, let's go with No and see if that causes problems in the wild (It didn't during my testing)